uploader/wiki_uploader.py

```diff
@@ -23,14 +23,14 @@ def __init__(
             access_secret=access_secret,
         )
 
-    def upload_file(self, file_name, file_stream, description="", license=""):
+    def upload_file(self, file_name, file_stream, date_created="", description="", license=""):
         if not description:
             description = file_name
 
         upload_result = self.mw_client.upload(
             file=file_stream,
             filename=file_name,
-            description=get_initial_page_text(license, description),
+            description=get_initial_page_text(license, date_created, description),
```

Comment:
okey, so we have a problem here. This function `get_initial_page_text` has a different order of arguments. One way would be to send in named arguments. for eg: 
```
        upload_result = self.mw_client.upload(
            file=file_stream,
            filename=file_name,
            description=get_initial_page_text(
                license=license,
                date_of_creation=date_created,
                summary=description
            ),
            ignore=True,
            comment=description,
        )
``` 

otherwise, this was giving wrong results.